src/server/pachyderm_test.go

```diff
@@ -1020,6 +1020,14 @@ func TestProvenance(t *testing.T) {
 	commitInfos, err = c.ListCommit(cPipeline, "master", "", 0)
 	require.NoError(t, err)
 	require.Equal(t, 2, len(commitInfos))
+
```

Comment:
Would it be possible to move this into its own test? I know TestProvenance is already pretty far gone, but in the past I've had bad experiences where we need to fix or remove complicated tests because we've changed our semantics, and we end up removing a bunch of auxiliary tests (like this one) that should still apply. If tests are smaller and more well-defined, then changing semantics without introducing regressions is easier

Reply:
Done.